templates/vue/src/components/NodeModal.vue

```diff
@@ -287,6 +326,9 @@ export default {
       }
       return this.hasDraftPermission(ID)
     },
+    tapestryEditor() {
```

Comment:
All methods and calculated values should start with a verb for clarity. This should read `canEditTapestry`.

Reply:
Done